### PR TITLE
add log scale

### PR DESCRIPTION
**Pull request by anastasia-escher**
solves #462 

only log scale for now

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,7 +76,7 @@
   "scripts": {
     "start": "craco start",
     "build": "craco build",
-    "test": "craco test",
+    "test": "craco test -u",
```
Comment (chaoran-chen) [inline package.json]: Please don't change the default behavior of the test command.

**Review by chaoran-chen — CHANGES_REQUESTED** [review]
Hi @anastasia-escher, thanks for the PR! The absolute plot is not correct. When you check the log scale option and switch to the "absolute" numbers, the log scale checkbox disappears but it seems that it is still on and the y-axis is wrong:

image

**Review by chaoran-chen — CHANGES_REQUESTED** [review]
If I switch to the "Absolute" view and then switch back to "Proportion", the plot looks like this:

image

(Observed in both Chrome and Firefox on Windows)

**Comment by chaoran-chen** [comment]
Hmm.. this also isn't ideal... there is now a lot of empty space:

image

Why is the problem occurring?

**Comment by anastasia-escher** [comment]
> Hmm.. this also isn't ideal... there is now a lot of empty space:
> 
> image
> 
> Why is the problem occurring?

This has to do with the height of the plot wrapper and the margin-bottom of the plot... I'll try to make it look nicer.

**Review by chaoran-chen — CHANGES_REQUESTED** [review]
In the compare mode, the plot is now very small. The Log scale checkbox should be placed in the same row as the show CI button if there is sufficient space. There also shouldn't be so much empty space at to bottom.

image

**Comment by corneliusroemer** [comment]
This is nice already, though as @chaoran-chen mentioned, the button could be placed in the same row as `absolute` etc
image

**Review by chaoran-chen — CHANGES_REQUESTED** [review]
The UI is unfortunately still not working properly.

On the mobile view, it's plot is still tiny:

image

It introduced a lot of space under many plots. This is how the growth advantage plot looks on the production page:

image


This is how this PR looks like:

image

**Comment by chaoran-chen** [comment]
@dameyerdave, would you have the time to take a look and help out here?

File: src/widgets/common.tsx [hunk 2]
```diff
@@ -40,6 +40,7 @@ export const ChartAndMetricsWrapper = styled.div`
   display: flex;
   flex-direction: column;
   flex: 1;
+  margin-bottom: 70px;
```
Comment (chaoran-chen) [inline src/widgets/common.tsx]: This line looks problematic. The `ChartAndMetricsWrapper` is a widely used component. Is it a good idea to set such a fixed margin here?

**Comment by dameyerdave** [comment]
Hi Chaoran, I totally missed you question. I'm very sorry for that. @anastasia-escher is on this and will solve this.

**Review by chaoran-chen — APPROVED** [review]
Thanks!

**Pull request merged**